### PR TITLE
Adding GL codes to gov accounts

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6708

*Description of changes:*
- Add revenueAccount to paymentInfo schema.
- Add distribution codes against the payment account

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

**Review by mengdong19 — APPROVED** [review]

File: pay-api/src/pay_api/models/distribution_code.py [hunk 1]
```diff
@@ -66,23 +66,28 @@ class DistributionCode(Audit):  # pylint:disable=too-many-instance-attributes
...
     @classmethod
-    def find_all(cls):
+    def find_all(cls, include_gov_account_gl_codes: bool = False):
         """Find all distribution codes."""
         valid_date = date.today()
         query = cls.query.filter(DistributionCode.start_date <= valid_date). \
             filter((DistributionCode.end_date.is_(None)) | (DistributionCode.end_date >= valid_date)). \
             order_by(DistributionCode.name.asc())
 
+        query = query.filter(DistributionCode.account_id.isnot(None)) if include_gov_account_gl_codes \
```
Comment (saravanpa-aot) [inline pay-api/src/pay_api/models/distribution_code.py:88]: so gov_account_gl_codes needs account_id ..
may be write a tiny one line comment , will be helpful

File: pay-api/src/pay_api/services/ejv_pay_service.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+class EjvPayService(PaymentSystemService, OAuthService):
+    """Service to manage EJV pay accounts."""
+
+    def get_payment_system_code(self):
+        """Return CGI as the system code."""
+        return PaymentSystem.CGI.value
+
+    def get_payment_method_code(self):
+        """Return CC as the method code."""
```
Comment (saravanpa-aot) [inline pay-api/src/pay_api/services/ejv_pay_service.py:39]: comment says CC

**Review by saravanpa-aot — APPROVED** [review]

File: pay-api/src/pay_api/models/distribution_code.py (hunk 1 above)
Comment (sumesh-aot) [reply pay-api/src/pay_api/models/distribution_code.py:89]: Done

File: pay-api/src/pay_api/services/ejv_pay_service.py (hunk 2 above)
Comment (sumesh-aot) [reply pay-api/src/pay_api/services/ejv_pay_service.py]: Fixed

**Pull request merged**